subprojects/build-comparison/src/main/java/org/gradle/api/plugins/buildcomparison/gradle/internal/DefaultGradleBuildInvocationSpec.java

```diff
@@ -34,6 +35,7 @@
     private List<String> tasks = new LinkedList<String>();
     private List<String> arguments = new LinkedList<String>();
 
+    @Inject
```

Comment:
Thanks for clarifying. @eriwen Can you still add that to the Javadocs of the `ObjectFactory`?